sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1605,3 +1605,103 @@ private case class GetTimestamp(
   override def withTimeZone(timeZoneId: String): TimeZoneAwareExpression =
     copy(timeZoneId = Option(timeZoneId))
 }
+
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(year, month, day, hour, min, sec, timezone) - Create timestamp from year, month, day, hour, min, sec and timezone fields.",
+  arguments = """
+    Arguments:
+      * year - the year to represent, from 1 to 9999
+      * month - the month-of-year to represent, from 1 (January) to 12 (December)
+      * day - the day-of-month to represent, from 1 to 31
+      * hour - the hour-of-day to represent, from 0 to 23
+      * minute - the minute-of-hour to represent, from 0 to 59
+      * second  the second-of-minute and its micro-fraction to represent, from
+      *         0 to 59.999999
+  """,
```

Comment:
For a record, PostgreSQL works like the following.
```
For a record, PostgreSQL `second` can be `60`.
```
postgres=# select make_timestamp(1,1,1,0,0,60);
   make_timestamp
---------------------
 0001-01-01 00:01:00
(1 row)
```

Reply:
I would support the feature separately. I will open an JIRA ticket for that.

Reply:
60 seconds have been supported